cmd/nginx/flags.go

```diff
@@ -171,6 +171,8 @@ Takes the form "<host>:port". If not provided, no admission controller is starte
 		profilerPort = flags.Int("profiler-port", 10245, "Port to use for expose the ingress controller Go profiler when it is enabled.")
 
 		statusUpdateInterval = flags.Int("status-update-interval", status.UpdateInterval, "Time interval in seconds in which the status should check if an update is required. Default is 60 seconds")
+
+		shutdownGracePeriod = flags.Int("shutdown-grace-period", 0, "Seconds to wait after receiving the shutdown signal, before stopping the nginx process.")
```

Comment:
Does the helm templates need to be updated as well so we can customize this value?

Reply:
@kolorful no need, you can use [extraArgs](https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/values.yaml#L120) to set custom flags or values

Reply:
ah nice.